### ISSUE TITLE
reach out to zerocopy folks

### ISSUE THREAD

**Issue by cmyr** [googlefonts/oxidize#16]
In this preliminary work I've been using the [zerocopy](https://docs.rs/zerocopy/latest/zerocopy/index.html) crate extensively. This is working well, but unfortunately there is no public bug tracker, and I'd like to figure out a way for us to communicate certain small issues or questions to that crates maintainers.

A lot of these are small things; for instance there are some methods that are missing `#[inline]` annotations in the upstream and this can cause us to generate suboptimal code.

The principal maintainer  (to the best of my knowledge) is @joshlf. I sent them an email earlier in the week, but maybe it's worth tracking this effort here.

Basically: at the very least I would like some informal mechanism to propose and discuss small changes with that team (which could be as simple as opening issues in this repo and tagging josh, but really anything that achieves that goal is fine.)

Ultimately we need to have really fine-grained control over the code that we're generating, and if we aren't able to make upstream changes at all then we'll have to consider another option; for me this would most likely mean maintaining a fork, which is fine but not desirable.

**Comment by joshlf** [comment]
We use Fuchsia's [public bug tracker](https://fxbug.dev) (it's not well-documented, which is something we need to improve). Use the `Rust>lib>zerocopy` component for issues relating to zerocopy. Email is always fine too for things which you'd just like to discuss - I'm just catching up on email because I was out for much of last week.

**Comment by cmyr** [comment]
@joshlf Awesome, thanks, I (clearly) didn't know about that. :)
